examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -1973,6 +1977,29 @@ void handleNotFound(void) {
   server.send(404, "text/plain", message);
 }
 
+#if MQTT_SERVER_AUTODETECT_ENABLE
+void mqtt_detect_server(void) {
+  debug("Querying MQTT server...");
```

Comment:
Change "Querying" to "Auto detecting" or "Finding the" or "Looking for the" perhaps?